src/Directory.Build.props

```diff
@@ -0,0 +1,35 @@
+<Project>
```

Comment:
This file was not in the list. I've raised a [question](https://github.com/Particular/NServiceBus.AzureStorageQueues/pull/247#issuecomment-324814784) to confirm we actually need it. Most likely we do.

Reply:
It is in the list in https://github.com/Particular/DotNetCoreLaunch/issues/7, and we have added it to every other conversion project.

Reply:
I've copied the steps to ASQ PR and after that `Directory.Build.props` item was added. Ignore my comment. 😃